problems/arraysIntro.js

```diff
@@ -64,29 +83,46 @@ function arrayToString() {}
  * @param {Array} array2
  * @returns {string|Array}
  */
-function largerArray() {}
+function largerArray(array1, array2) {
+  if (array1.length > array2.length) {
+    return array1;
+  } else if (array1.length < array2.length) {
+    return array2;
+  } else {
+    return "They are the same size";
+  }
+}
 
 /**
  * Takes in an array and returns the middle element.
  * If the array is even in length return the middle two elements in their own array.
  * @param {Array} array
  * @returns {*|Array}
- */
-function middleElement() {}
+ */ //[1,2,3,4] array.slice(1,3) => [2,3] 
+function middleElement(array) {
+  if (array.length % 2 === 1) {
+    return array[(array.length - 1) / 2]
+  } else {
+    return array.slice(Math.floor((array.length - 1) / 2), Math.floor((array.length - 1) / 2) + 2)
```

Comment:
This is great but try storing these math values in variables so you can make cleaner return statements